cmd/kubeadm/app/preflight/checks_test.go

```diff
@@ -849,3 +850,29 @@ func TestNumCPUCheck(t *testing.T) {
 		})
 	}
 }
+
+func TestMemCheck(t *testing.T) {
+	// skip this test, if OS in not Linux, since it will ONLY pass on Linux.
+	if runtime.GOOS != "linux" {
+		return
```

Comment:
Instead of `return` a `t.Skip("unsupported OS for memory check test)` should be clearer?

Reply:
done